### ISSUE TITLE
Not building on Mac OS X 10.12

### ISSUE THREAD

**Issue by madronalabs** [wrl/rutabaga#6]
Running ./waf, I get
```[ 68/107] c: src/element.c -> build/src/element.c.8.o
../src/platform/cocoa/window.m:413:3: error: 'NSTitledWindowMask' is deprecated: first deprecated in macOS 10.12 [-Werror,-Wdeprecated-declarations]
                NSTitledWindowMask
                ^~~~~~~~~~~~~~~~~~
                NSWindowStyleMaskTitled
/System/Library/Frameworks/AppKit.framework/Headers/NSWindow.h:953:32: note: 'NSTitledWindowMask' has been explicitly marked deprecated here
static const NSWindowStyleMask NSTitledWindowMask API_DEPRECATED_WITH_REPLACEMENT("NSWindowStyleMaskTitled", macosx(10.0, 10.12)) = NSWindowStyleMaskTitled;
                               ^
```

followed by a lot of similar deprecation errors.

**Comment by madronalabs** [comment]
I see there is already a pull request about this. It looks like that change will require SDK 10.12+. As an alternative, simply adding -Wno-deprecated-declarations to CFLAGS will allow the old symbols to compile until they are truly EOL.

**Comment by wrl** [comment]
`-Wno-deprecated-declarations` looks like a good stopgap. I'll find some time soon to modify that pull request such that the new constants are behind some `#if` blocks for checking the version of the MacOS SDK rtb is being built against.

**Comment by wrl** [comment]
fixed for good in PR #5
